### PR TITLE
Clang-format test (DO NOT SUBMIT)

### PR DESCRIPTION
**Pull request by aheejin**
This tests two things:
- What remotes and branches does Travis CI binaryen have
- Whether clang-format error will be correctly detected

### CODE REVIEW

File: clang-format-diff.sh [hunk 1]
```diff
@@ -3,6 +3,7 @@
 
 # For testing
 git br -v
```
Comment (sbc100) [inline clang-format-diff.sh]: I think `br` is your local alias for `branch` so it won't work here.

Comment (aheejin) [reply clang-format-diff.sh]: Oh right omg

**Comment by aheejin** [comment]
According to this result: https://travis-ci.org/WebAssembly/binaryen/jobs/539951146
It looks like binaryen repo in Travis CI has both `master` and `origin/master`, and the clang-format hook works. Not sure why your first try at `origin/master` failed though.

Travis CI stdout:
```
$ ./clang-format-diff.sh
* (HEAD detached at FETCH_HEAD) d73d2d4 Merge 1d0d92e39aa36f32ff3b6220646be85a2056d56e into c5b50feac3e1faeb51c6910e37357596e4700717
  master                        c5b50fe [origin/master] Fix run_gcc_tests with out-of-tree builds (#2152)
origin	https://github.com/WebAssembly/binaryen.git (fetch)
origin	https://github.com/WebAssembly/binaryen.git (push)
Run git clang-format before committing!
diff --git a/src/wasm/wasm.cpp b/src/wasm/wasm.cpp
index 260e242..5e54451 100644
--- a/src/wasm/wasm.cpp
+++ b/src/wasm/wasm.cpp
@@ -1072,8 +1072,7 @@ void Module::updateMaps() {
   }
 
   // Deliberately introduce a clang-format error for testing
-  int
-      a = 3;
+  int a = 3;
   (void)a;
```

**Comment by aheejin** [comment]
Resolved by #2156.

**Pull request closed without merging**